src/Kunstmaan/NodeBundle/Resources/views/NodeAdmin/Modals/_pagetemplate.html.twig

```diff
@@ -0,0 +1,34 @@
+{% import _self as macros %}
+
+{% for row in pageTemplate.rows %}
+<div class="row">
```

Comment:
Hi @aistis- 

This will never give back to correct representation of the grid system.

My twig now returns:

`<div class="row">
    <div class="col-md-3"></div>
    <div class="col-md-9">
    <div class="row">
        <div class="col-md-6"></div>
        <div class="col-md-6"></div>
    </div>
    <div class="row">
        <div class="col-md-12"></div>
    </div>
</div>`

How do you think this will do the same with the whitespace control modifier?